template-parts/header_content-title_subtitle.php

```diff
@@ -0,0 +1,33 @@
+<?php
+$obj           = ucfwp_get_queried_object();
+$title         = ucfwp_get_header_title( $obj );
+$subtitle      = ucfwp_get_header_subtitle( $obj );
+$h1            = ucfwp_get_header_h1_option( $obj );
+$h1_elem       = ucfwp_get_header_h1_elem( $obj );
+$title_elem    = ( $h1 === 'title' ) ? $h1_elem : 'span';
+$subtitle_elem = ( $h1 === 'subtitle' ) ? $h1_elem : 'span';
+
+$clusters_link = get_theme_mod( 'clusters_list_page' ) ? get_permalink( get_theme_mod( 'clusters_list_page' ) ) : null;
+?>
+
+<?php if ( $title ): ?>
+	<div class="header-content-inner align-self-start pt-4 pt-sm-0 align-self-sm-center">
+		<div class="container">
+			<?php if ( $clusters_link ) : ?>
```

Comment:
Do we want this link showing up on all pages with a header bg and not-custom header contents?  There doesn't seem to be anything in this code that limits what pages this link appears on.